edx_proctoring/static/proctoring/spec/proctored_exam_spec.js

```diff
@@ -0,0 +1,76 @@
+describe('ProctoredExamView', function () {
+    beforeEach(function () {
+        this.server = sinon.fakeServer.create();
+        jasmine.clock().install();
+        setFixtures(
+            '<div class="proctored_exam_status">' +
+            '<script type="text/template" id="proctored-exam-status-tpl">' +
+            '<div class="exam-timer">' +
+            '<%- gettext("You are taking") %>' +
+            '<a href="<%= exam_url_path %>"> <%= exam_display_name %> </a>' +
+            '<%- gettext(" exam as a proctored exam. Good Luck!") %>' +
```

Comment:
@chrisndodge Edits from an earlier comment that did not make it:
1) Put quotation marks around the name of the exam.
2) Strike the occurrence of "exam" after the inserted exam name.
3) Remove "Good Luck!" 
The updated string should read, ""You are taking {Exam_display_name} as a proctored exam. The timer on the right shows the time remaining in the exam."